<!doctype html>
<html class="no-js" lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>
    
  2023-11-1 journal scan - 
  
  </title>
  
  
  <link href="atom.xml" rel="alternate" title="" type="application/atom+xml">
    <link rel="stylesheet" href="asset/css/foundation.min.css" />
    <link rel="stylesheet" href="asset/css/docs.css" />
    <script src="asset/js/vendor/modernizr.js"></script>
    <script src="asset/js/vendor/jquery.js"></script>
 
<script type="text/javascript">
  function before_search(){
    var searchVal = 'site: ' + document.getElementById('search_input').value;
    document.getElementById('search_q').value = searchVal;
    return true;
  }
</script>
  </head>
  <body class="antialiased hide-extras">
    
    <div class="marketing off-canvas-wrap" data-offcanvas>
      <div class="inner-wrap">


<nav class="top-bar docs-bar hide-for-small" data-topbar>


  <section class="top-bar-section">
  <div class="row">
      <div style="position: relative;width:100%;"><div style="position: absolute; width:100%;">
        <ul id="main-menu" class="left">
        
        <li id=""><a target="_self" href="index.html">Home</a></li>
        
        <li id=""><a target="_self" href="archives.html">Archives</a></li>
        
        <li id=""><a target="_self" href="JournalScan.html">JournalScan</a></li>
        
        </ul>

        <ul class="right" id="search-wrap">
          <li>
<form target="_blank" onsubmit="return before_search();" action="https://google.com/search" method="get">
    <input type="hidden" id="search_q" name="q" value="" />
    <input tabindex="1" type="search" id="search_input"  placeholder="Search"/>
</form>
</li>
          </ul>
      </div></div>
  </div>
  </section>

</nav>

        <nav class="tab-bar show-for-small">
  <a href="javascript:void(0)" class="left-off-canvas-toggle menu-icon">
    <span> &nbsp; </span>
  </a>
</nav>

<aside class="left-off-canvas-menu">
      <ul class="off-canvas-list">
        
        <li><a target="_self" href="index.html">Home</a></li>
        
        <li><a target="_self" href="archives.html">Archives</a></li>
        
        <li><a target="_self" href="JournalScan.html">JournalScan</a></li>
        

    <li><label>Categories</label></li>

         

      </ul>
    </aside>

<a class="exit-off-canvas" href="#"></a>


        <section id="main-content" role="main" class="scroll-container">
        
       

 <script type="text/javascript">
	$(function(){
		var currentURL = '2023-11-1%20journal%20scan.html';
		currentURL = currentURL.substr(0,currentURL.length-5);
		$('#menu_item_'+currentURL).addClass('is_active');
	});
</script>
<div class="row">

    <div id="single-page-wrap">
        <h1>2023-11-1 journal scan</h1>

        <div class="markdown-body post-page">
        <p>author: Qiannzzz</p>
<div class="mweb_toc"><ul>
<li><a href="#1-nest-nested-hierarchical-structure-identification-in-spatial-transcriptomic-data">1- NeST: nested hierarchical structure identification in spatial transcriptomic data</a></li>
<li><a href="#2-decoding-spatial-organization-maps-and-context-specific-landscapes-of-breast-cancer-and-its-microenvironment-via-high-resolution-spatial-transcriptomic-analysis">2-Decoding spatial organization maps and context-specific landscapes of breast cancer and its microenvironment via high-resolution spatial transcriptomic analysis</a></li>
<li><a href="#3-graph-attention-based-fusion-of-pathology-images-and-gene-expression-for-prediction-of-cancer-survival">3-Graph attention-based fusion of pathology images and gene expression for prediction of cancer survival</a></li>
<li><a href="#4-infernape-uncovers-cell-type-specific-and-spatially-resolved-alternative-polyadenylation-in-the-brain">4-Infernape uncovers cell type-specific and spatially resolved alternative polyadenylation in the brain</a></li>
<li><a href="#5-cell-segmentation-and-gene-imputation-for-imaging-based-spatial-transcriptomics">5-Cell segmentation and gene imputation for imaging-based spatial transcriptomics</a></li>
<li><a href="#6-mapping-the-spatial-transcriptomic-signature-of-the-hippocampus-during-memory-consolidation">6-Mapping the spatial transcriptomic signature of the hippocampus during memory consolidation</a></li>
<li><a href="#7-latent-feature-extraction-with-a-prior-based-self-attention-framework-for-spatial-transcriptomics">7-Latent feature extraction with a prior-based self-attention framework for spatial transcriptomics</a></li>
<li><a href="#8-programs-origins-and-niches-of-immunomodulatory-myeloid-cells-in-gliomas">8-Programs, Origins, and Niches of Immunomodulatory Myeloid Cells in Gliomas **</a></li>
<li><a href="#9-revealing-molecular-mechanisms-of-early-onset-tongue-cancer-by-spatial-transcriptomics">9-Revealing Molecular Mechanisms of Early-Onset Tongue Cancer by Spatial Transcriptomics</a></li>
<li><a href="#10-digital-profiling-of-cancer-transcriptomes-from-histology-images-with-grouped-vision-attention">10-Digital profiling of cancer transcriptomes from histology images with grouped vision attention</a></li>
<li><a href="#11-st-assign-a-probabilistic-model-for-joint-cell-type-identification-in-spatial-transcriptomics-and-single-cell-rna-sequencing-data">11-ST-Assign: a probabilistic model for joint cell type identification in spatial transcriptomics and single-cell RNA sequencing data</a></li>
<li><a href="#12-controlling-organoid-gene-expression-with-light">12-Controlling organoid gene expression with light</a></li>
<li><a href="#13-automated-bioinformatics-analysis-via-autoba">13-Automated Bioinformatics Analysis via AutoBA</a></li>
<li><a href="#14-a-transient-wave-of-bhlhe41-resident-macrophages-enables-remodeling-of-the-developing-infarcted-myocardium-load">14-A transient wave of Bhlhe41+ resident macrophages enables remodeling of the developing infarcted myocardium  load</a></li>
<li><a href="#15-spatiotemporal-optogenetic-control-of-gene-expression-in-organoids">15-Spatiotemporal, optogenetic control of gene expression in organoids []</a></li>
<li><a href="#16-insitucor-a-toolkit-for-discovering-non-trivial-spatial-correlations-in-spatial-transcriptomics">16-InSituCor: a toolkit for discovering non-trivial spatial correlations in spatial transcriptomics</a></li>
<li><a href="#17-moving-closer-toward-a-comprehensive-view-of-tumor-biology-and-microarchitecture-using-spatial-transcriptomics">17-Moving closer toward a comprehensive view of tumor biology and microarchitecture using spatial transcriptomics ***</a></li>
<li><a href="#18-scan-spatiotemporal-cloud-atlas-for-neural-cells">18-SCAN: Spatiotemporal Cloud Atlas for Neural cells</a></li>
<li><a href="#19-tissue-uncertainty-calibrated-prediction-of-single-cell-spatial-transcriptomics-improves-downstream-analyses">19-TISSUE: uncertainty-calibrated prediction of single-cell spatial transcriptomics improves downstream analyses</a></li>
<li><a href="#20-spatial-multimodal-analysis-of-transcriptomes-and-metabolomes-in-tissues">20-Spatial multimodal analysis of transcriptomes and metabolomes in tissues</a></li>
<li><a href="#21-sc2spa-a-deep-learning-based-approach-to-map-transcriptome-to-spatial-origins-at-cellular-resolution">21-SC2Spa: a deep learning-based approach to map transcriptome to spatial origins at cellular resolution</a></li>
<li><a href="#22-characterizing-tissue-structures-from-spatial-omics-with-spatial-cellular-graph-partition">22-Characterizing tissue structures from spatial omics with spatial cellular graph partition</a></li>
<li><a href="#23-ficture-scalable-segmentation-free-analysis-of-submicron-resolution-spatial-transcriptomics">23-FICTURE: Scalable segmentation-free analysis of submicron resolution spatial transcriptomics</a></li>
<li><a href="#24-principles-and-challenges-of-modeling-temporal-and-spatial-omics-data-down">24-Principles and challenges of modeling temporal and spatial omics data *** down</a></li>
<li><a href="#25-circular-extrachromosomal-dna-promotes-tumor-heterogeneity-in-high-risk-medulloblastoma">25-Circular extrachromosomal DNA promotes tumor heterogeneity in high-risk medulloblastoma</a></li>
<li><a href="#26-the-power-and-potential-of-mitochondria-transfer">26-The power and potential of mitochondria transfer</a></li>
<li><a href="#27-transcriptional-and-epigenetic-regulators-of-human-cd8-t-cell-function-identified-through-orthogonal-crispr-screens">27-Transcriptional and epigenetic regulators of human CD8+ T cell function identified through orthogonal CRISPR screens</a></li>
</ul>
</div>
<h4><a id="1-nest-nested-hierarchical-structure-identification-in-spatial-transcriptomic-data" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>1- NeST: nested hierarchical structure identification in spatial transcriptomic data</h4>
<p>nat com  Qing Nie-the University of California Irvine<br />
software<br />
gap: traditional methods use max similarity to disjoint regions,<br />
advantage: this method can discover region-within-region structure,  <br />
path: coexpression hotspots—regions exhibiting localized spatial coexpression of some set of genes<br />
<figure><img src="media/16995006052889/16995007334137.jpg" alt="" /></figure></p>
<h4><a id="2-decoding-spatial-organization-maps-and-context-specific-landscapes-of-breast-cancer-and-its-microenvironment-via-high-resolution-spatial-transcriptomic-analysis" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>2-Decoding spatial organization maps and context-specific landscapes of breast cancer and its microenvironment via high-resolution spatial transcriptomic analysis</h4>
<p>bioRxiv woong-Yang Park<br />
key: breast cancer; image-based st, TME<br />
highlight immune cell gradients transitioning from the tumor periphery to its core regions and from tertiary lymphoid structure to immune inflamed regions in alignment with the specific function of each cell type.<br />
<figure><img src="media/16995006052889/16995006794034.jpg" alt="" /></figure></p>
<h4><a id="3-graph-attention-based-fusion-of-pathology-images-and-gene-expression-for-prediction-of-cancer-survival" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>3-Graph attention-based fusion of pathology images and gene expression for prediction of cancer survival</h4>
<p>bioRxiv Vijaya B Kolachalama<br />
key: cancer survival prediction;<br />
gap: most frameworks for multimodal data fusion do not fully account for the interactions between different modalities<br />
combine gene expression and histology into a model, use histology as an undirected graph via machine learning</p>
<h4><a id="4-infernape-uncovers-cell-type-specific-and-spatially-resolved-alternative-polyadenylation-in-the-brain" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>4-Infernape uncovers cell type-specific and spatially resolved alternative polyadenylation in the brain</h4>
<p>Genome research Xiaochang Zhang software-Infernape<br />
background: differential polyadenylation sites regulate gene expression cell-type specific<br />
usage: can infer quantified PA usage from a single cell and spatial transcriptomic data<br />
results: intronic PAs and 3’UTR lengthening during cortical neurogenesis</p>
<h4><a id="5-cell-segmentation-and-gene-imputation-for-imaging-based-spatial-transcriptomics" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>5-Cell segmentation and gene imputation for imaging-based spatial transcriptomics</h4>
<p>bioRvix Xiawen Ren software-RedeFISH<br />
usage: cell segment and gene imputation<br />
conduct cell segmentation and gene imputation for imaging-based spatial transcriptomics with the aid of single-cell RNA sequencing data</p>
<h4><a id="6-mapping-the-spatial-transcriptomic-signature-of-the-hippocampus-during-memory-consolidation" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>6-Mapping the spatial transcriptomic signature of the hippocampus during memory consolidation</h4>
<p>nature communications Snehajyoti Chatterjee<br />
key: ST, hippo memory<br />
back: memory consolidation involves discrete patterns of transcriptional events in hippo;<br />
result:  CA1 subregion-specific expression of a transcription factor subfamily has a critical functional role in the consolidation of long-term memory</p>
<h4><a id="7-latent-feature-extraction-with-a-prior-based-self-attention-framework-for-spatial-transcriptomics" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>7-Latent feature extraction with a prior-based self-attention framework for spatial transcriptomics</h4>
<p>genome research, Shengquan Chen nankai.edu.cn software-PAST<br />
introduction: a prior-based self-attention framework for spatial transcriptomics, a variational graph convolutional autoencoder, integrates prior information via a Bayesian neural network, captures spatial patterns via a self-attention mechanism, and enables scalable application via a ripple walk sampler strategy.<br />
features: past is the first ST method that integrates reference data to analyze ST data</p>
<h4><a id="8-programs-origins-and-niches-of-immunomodulatory-myeloid-cells-in-gliomas" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>8-Programs, Origins, and Niches of Immunomodulatory Myeloid Cells in Gliomas **</h4>
<p>bioRvix Bradley E. Bernstein-Harvard Medical School<br />
Data: 183k myeloid cells express four immunomodulatory activity programs<br />
Scavenger Immunosuppressive, C1Q Immunosuppressive,CXCR4 Inflammatory, and IL1B 		Inflammatory All four programs are present in IDH1 mutant <br />
multi-modality: scRNA-seq data and mitochondrial DNA-based lineage tracing, spatial transcriptomics, and organoid explant systems <br />
result: these programs are driven by microenvironment and therapy, not cell type, origin, or mutant status.<br />
C1Q Immunosuppressive program- routinely administered dexamethasone<br />
Scavenger Immunosuppressive program includes ligands with established roles in T-cell suppression, are induced in hypoxic regions, and immunotherapy resistance.</p>
<h4><a id="9-revealing-molecular-mechanisms-of-early-onset-tongue-cancer-by-spatial-transcriptomics" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>9-Revealing Molecular Mechanisms of Early-Onset Tongue Cancer by Spatial Transcriptomics</h4>
<p>bioRvix Evgeny V. Denisov-Russian National Research Center<br />
methods: spatial whole transcriptome<br />
obj: early tongue cancer compared with old one<br />
finding:</p>
<ol>
<li>oxidative stress, vascular mimicry, MAPK, and JAK-STAT pathways,</li>
<li>myeloid-derived suppressor cells, TAM, and plasma cells increased.</li>
<li>the invasive front with vascular mimicry with the arrangement of TAM and aggregation of plasma cells and lymphocytes organized into a teritary lymphoid structure.</li>
<li>Early-onset tongue cancer demonstrates extremely downregulated oxidative phosphorylation and upregulated glycolysis.</li>
</ol>
<h4><a id="10-digital-profiling-of-cancer-transcriptomes-from-histology-images-with-grouped-vision-attention" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>10-Digital profiling of cancer transcriptomes from histology images with grouped vision attention</h4>
<p>bioRvix Olivier Gevaert-stanford uni software-SEQUOIA<br />
description: a deep learning model employing a transformer architecture to predict cancer transcriptomes from whole slide histology images.<br />
Data: pre-train 2242 normal and 4218 tumor<br />
histology? to predict gene expression?</p>
<h4><a id="11-st-assign-a-probabilistic-model-for-joint-cell-type-identification-in-spatial-transcriptomics-and-single-cell-rna-sequencing-data" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>11-ST-Assign: a probabilistic model for joint cell type identification in spatial transcriptomics and single-cell RNA sequencing data</h4>
<p>bioRvix Ewa Szczurek Mechanics, University of Warsaw software ST-align<br />
description: joint sc data type annotation and mixture decomposition in st data to enhance cell-type identification, account for shared variables such as gene expression profile, and leverage prior knowledge about marker genes. this model and a generative graph-based structure and employ Markov chain Monte Carlo for the inference.</p>
<h4><a id="12-controlling-organoid-gene-expression-with-light" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>12-Controlling organoid gene expression with light</h4>
<p>nature review genetics. Henry Ertl technology<br />
combines optogenetics and genetic perturbation technology in organoids to control the expression of endogenous genes using light.</p>
<h4><a id="13-automated-bioinformatics-analysis-via-autoba" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>13-Automated Bioinformatics Analysis via AutoBA</h4>
<p>bioRvix Xin Gao Kingdom of Saudi Arabia software AutoBA<br />
Description: an autonomous AI agent designed explicitly for regular omics data analysis based on large language models</p>
<h4><a id="14-a-transient-wave-of-bhlhe41-resident-macrophages-enables-remodeling-of-the-developing-infarcted-myocardium-load" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>14-A transient wave of Bhlhe41+ resident macrophages enables remodeling of the developing infarcted myocardium  load</h4>
<p>Cell Reports,  Yaozu Xiang, Shanghai East Hosptial</p>
<h4><a id="15-spatiotemporal-optogenetic-control-of-gene-expression-in-organoids" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>15-Spatiotemporal, optogenetic control of gene expression in organoids []</h4>
<p>Nature Methods  Nikolaus Rajewsky<br />
optogenetics and gene perturbation technologies to activate or knock down RNA of target genes in programmable spatiotemporal patterns</p>
<h4><a id="16-insitucor-a-toolkit-for-discovering-non-trivial-spatial-correlations-in-spatial-transcriptomics" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>16-InSituCor: a toolkit for discovering non-trivial spatial correlations in spatial transcriptomics</h4>
<p>bio Joseph M Beechem<br />
Des: software to find the correlation of spatial genes, such as ligand and receptor</p>
<h4><a id="17-moving-closer-toward-a-comprehensive-view-of-tumor-biology-and-microarchitecture-using-spatial-transcriptomics" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>17-Moving closer toward a comprehensive view of tumor biology and microarchitecture using spatial transcriptomics ***</h4>
<p>Nature Communications, De-Chen Lin, <em>comments</em></p>
<h4><a id="18-scan-spatiotemporal-cloud-atlas-for-neural-cells" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>18-SCAN: Spatiotemporal Cloud Atlas for Neural cells</h4>
<p>Nucleic Acids Research, Dongsheng Chen<br />
Des: a dataset of neuron cells</p>
<h4><a id="19-tissue-uncertainty-calibrated-prediction-of-single-cell-spatial-transcriptomics-improves-downstream-analyses" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>19-TISSUE: uncertainty-calibrated prediction of single-cell spatial transcriptomics improves downstream analyses</h4>
<p>bio, James Zou<br />
gene imputation</p>
<h4><a id="20-spatial-multimodal-analysis-of-transcriptomes-and-metabolomes-in-tissues" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>20-Spatial multimodal analysis of transcriptomes and metabolomes in tissues</h4>
<p>NBT, Joakim Lundeberg<br />
spatial omics: histology, mass spectrometry imaging and spatial transcriptomics<br />
data: mRNA transcripts and low-molecular-weight metabolites</p>
<h4><a id="21-sc2spa-a-deep-learning-based-approach-to-map-transcriptome-to-spatial-origins-at-cellular-resolution" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>21-SC2Spa: a deep learning-based approach to map transcriptome to spatial origins at cellular resolution</h4>
<p>bio, Kyoung Jae Won<br />
combined SC data and ST data, deconvolution, SC2Spa successfully mapped scRNAseq even to various low-resolution Visium data</p>
<h4><a id="22-characterizing-tissue-structures-from-spatial-omics-with-spatial-cellular-graph-partition" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>22-Characterizing tissue structures from spatial omics with spatial cellular graph partition</h4>
<p>bio, Alexandro E. Trevino<br />
Robust partitionings of both spatial transcriptomics and proteomics datasets</p>
<h4><a id="23-ficture-scalable-segmentation-free-analysis-of-submicron-resolution-spatial-transcriptomics" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>23-FICTURE: Scalable segmentation-free analysis of submicron resolution spatial transcriptomics</h4>
<p>bioRxiv - Bioinformatics, Hyun Min Kang<br />
Description: a segmentation-free spatial factorization method that can handle transcriptome-wide data labeled with billions of submicron-resolution spatial coordinates</p>
<h4><a id="24-principles-and-challenges-of-modeling-temporal-and-spatial-omics-data-down" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>24-Principles and challenges of modeling temporal and spatial omics data *** down</h4>
<p>Nature Methods  Oliver Stegle <em>review</em></p>
<h4><a id="25-circular-extrachromosomal-dna-promotes-tumor-heterogeneity-in-high-risk-medulloblastoma" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>25-Circular extrachromosomal DNA promotes tumor heterogeneity in high-risk medulloblastoma</h4>
<p>Nature Genetics, Lukas Chavez<br />
Circular extrachromosomal DNA (ecDNA) in patient tumors is an important driver of oncogenic gene expression<br />
Patients with ecDNA-positive medulloblastoma were more than twice as likely to relapse and three times as likely to die within 5 years of diagnosis</p>
<h4><a id="26-the-power-and-potential-of-mitochondria-transfer" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>26-The power and potential of mitochondria transfer</h4>
<p>Nature, Jonathan R. Brestoff, perspective<br />
some cell types export their mitochondria for delivery to developmentally unrelated cell types, a process called intercellular mitochondria transfer</p>
<h4><a id="27-transcriptional-and-epigenetic-regulators-of-human-cd8-t-cell-function-identified-through-orthogonal-crispr-screens" class="anchor" aria-hidden="true"><span class="octicon octicon-link"></span></a>27-Transcriptional and epigenetic regulators of human CD8+ T cell function identified through orthogonal CRISPR screens</h4>
<p>nature genetics,  Charles A. Gersbach</p>

        </div>
    </div>
</div>              
 <div class="page-bottom clearfix">
  <div class="row">
   <p class="copyright">Copyright &copy; 2015
Powered by <a target="_blank" href="http://www.mweb.im">MWeb</a>,&nbsp; 
Theme used <a target="_blank" href="http://github.com">GitHub CSS</a>.</p>
  </div>
</div>

        </section>
      </div>
    </div>



  














<style type="text/css">
figure{margin: 0;padding: 0;}
figcaption{text-align:center;}

/* PrismJS 1.14.0
 http://prismjs.com/download.html#themes=prism&languages=markup+css+clike+javascript */
/**
 * prism.js default theme for JavaScript, CSS and HTML
 * Based on dabblet (http://dabblet.com)
 * @author Lea Verou
 */

code[class*="language-"],
pre[class*="language-"] {
    color: black;
    background: none;
    text-shadow: 0 1px white;
    font-family: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
    text-align: left;
    white-space: pre;
    word-spacing: normal;
    word-break: normal;
    word-wrap: normal;
    line-height: 1.5;
    
    -moz-tab-size: 4;
    -o-tab-size: 4;
    tab-size: 4;
    
    -webkit-hyphens: none;
    -moz-hyphens: none;
    -ms-hyphens: none;
    hyphens: none;
}

pre[class*="language-"]::-moz-selection, pre[class*="language-"] ::-moz-selection,
code[class*="language-"]::-moz-selection, code[class*="language-"] ::-moz-selection {
    text-shadow: none;
    background:#b3d4fc;
}

pre[class*="language-"]::selection, pre[class*="language-"] ::selection,
code[class*="language-"]::selection, code[class*="language-"] ::selection {
    text-shadow: none;
    background: #b3d4fc;
}

@media print {
    code[class*="language-"],
    pre[class*="language-"] {
        text-shadow: none;
    }
}

/* Code blocks */
pre[class*="language-"] {
    padding: 1em;
    margin: .5em 0;
    overflow: auto;
}

:not(pre) > code[class*="language-"],
pre[class*="language-"] {
    background: #F7F7F7;
}

/* Inline code */
:not(pre) > code[class*="language-"] {
    padding: .1em;
    border-radius: .3em;
    white-space: normal;
}

.token.comment,
.token.prolog,
.token.doctype,
.token.cdata {
    color: slategray;
}

.token.punctuation {
    color: #999;
}

.namespace {
    opacity: .7;
}

.token.property,
.token.tag,
.token.boolean,
.token.number,
.token.constant,
.token.symbol,
.token.deleted {
    color: #905;
}

.token.selector,
.token.attr-name,
.token.string,
.token.char,
.token.builtin,
.token.inserted {
    color: #690;
}

.token.operator,
.token.entity,
.token.url,
.language-css .token.string,
.style .token.string {
    color: #9a6e3a;
    background: hsla(0, 0%, 100%, .5);
}

.token.atrule,
.token.attr-value,
.token.keyword {
    color: #07a;
}

.token.function,
.token.class-name {
    color: #DD4A68;
}

.token.regex,
.token.important,
.token.variable {
    color: #e90;
}

.token.important,
.token.bold {
    font-weight: bold;
}
.token.italic {
    font-style: italic;
}

.token.entity {
    cursor: help;
}


pre[class*="language-"].line-numbers {
    position: relative;
    padding-left: 3.8em;
    counter-reset: linenumber;
}

pre[class*="language-"].line-numbers > code {
    position: relative;
    white-space: inherit;
}

.line-numbers .line-numbers-rows {
    position: absolute;
    pointer-events: none;
    top: 0;
    font-size: 100%;
    left: -3.8em;
    width: 3em; /* works for line-numbers below 1000 lines */
    letter-spacing: -1px;
    border-right: 1px solid #999;

    -webkit-user-select: none;
    -moz-user-select: none;
    -ms-user-select: none;
    user-select: none;

}

    .line-numbers-rows > span {
        pointer-events: none;
        display: block;
        counter-increment: linenumber;
    }

        .line-numbers-rows > span:before {
            content: counter(linenumber);
            color: #999;
            display: block;
            padding-right: 0.8em;
            text-align: right;
        }

</style>

  
    

    <script src="asset/js/foundation.min.js"></script>
    <script>
      $(document).foundation();
      function fixSidebarHeight(){
        var w1 = $('.markdown-body').height();
          var w2 = $('#sidebar').height();
          if (w1 > w2) { $('#sidebar').height(w1); };
      }
      $(function(){
        fixSidebarHeight();
      })
      $(window).load(function(){
          fixSidebarHeight();
      });
     
    </script>



  </body>
</html>
